src/Common/tests/Tests/System/IO/PathInternal.Windows.Tests.cs

```diff
@@ -6,81 +6,139 @@
 using System.IO;
 using Xunit;
 
-public class PathInternal_Windows_Tests
+namespace Tests.System.IO
 {
-    [Theory,
-        InlineData(PathInternal.ExtendedDevicePathPrefix, PathInternal.ExtendedDevicePathPrefix),
-        InlineData(@"Foo", @"Foo"),
-        InlineData(@"C:\Foo", @"\\?\C:\Foo"),
-        InlineData(@"\\.\Foo", @"\\.\Foo"),
-        InlineData(@"\\?\Foo", @"\\?\Foo"),
-        InlineData(@"\??\Foo", @"\??\Foo"),
-        InlineData(@"//?/Foo", @"//?/Foo"),
-        InlineData(@"\\Server\Share", PathInternal.UncExtendedPathPrefix + @"Server\Share")
-        ]
     [PlatformSpecific(TestPlatforms.Windows)]
-    public void EnsureExtendedPrefixTest(string path, string expected)
+    public class PathInternalTests_Windows
     {
-        Assert.Equal(expected, PathInternal.EnsureExtendedPrefix(path));
-    }
+        [Theory,
+            InlineData(@"\\?\", @"\\?\"),
+            InlineData(@"Foo", @"Foo"),
+            InlineData(@"C:\Foo", @"\\?\C:\Foo"),
+            InlineData(@"\\.\Foo", @"\\.\Foo"),
+            InlineData(@"\\?\Foo", @"\\?\Foo"),
+            InlineData(@"\??\Foo", @"\??\Foo"),
+            InlineData(@"//?/Foo", @"//?/Foo"),
+            InlineData(@"\\Server\Share", PathInternal.UncExtendedPathPrefix + @"Server\Share")
+            ]
+        public void EnsureExtendedPrefixTest(string path, string expected)
+        {
+            Assert.Equal(expected, PathInternal.EnsureExtendedPrefix(path));
+        }
 
-    [Theory,
-        InlineData(@"", false),
-        InlineData(@"\\?\", true),
-        InlineData(@"\??\", true),
-        InlineData(@"\\.\", false),
-        InlineData(@"\\?", false),
-        InlineData(@"\??", false),
-        InlineData(@"//?/", false),
-        InlineData(@"/??/", false)
-        ]
-    [PlatformSpecific(TestPlatforms.Windows)]
-    public void IsExtendedTest(string path, bool expected)
-    {
-        Assert.Equal(expected, PathInternal.IsExtended(path));
-    }
+        [Theory,
+            InlineData(@"", false),
+            InlineData(@"\\?\", true),
+            InlineData(@"\??\", true),
+            InlineData(@"\\.\", false),
+            InlineData(@"\\?", false),
+            InlineData(@"\??", false),
+            InlineData(@"//?/", false),
+            InlineData(@"/??/", false)
+            ]
```

Comment:
Most of the inline data is common with IsDeviceData. We should move it to a common function

Reply:
The inputs are the same, but the expected results are different. I don't think it would make much sense here.